tests/test-wrapper.sh

```diff
@@ -30,10 +30,10 @@ for feature in $(echo $1 | sed "s/^.*@\(.*\).wrap/\1/" | tr "," "\n"); do
         annotations)
             export USE_OCI_ANNOTATIONS=yes
             ;;
-        http)
+        https)
             export USE_HTTPS=yes
             ;;
-        https)
+        http)
```

Comment:
I guess this is in the wrong commit, no?

Reply:
Thanks for taking a look! - I've put things into the right commits now